common/dynamicconfig/constants.go

```diff
@@ -2037,6 +2037,8 @@ const (
 	// Allowed filters: DomainName,TasklistName,TasklistType
 	MatchingEnableClientAutoConfig
 
+	EnableNoSQLHistoryTaskDualWriteMode
```

Comment:
Nit: it's not super clear from the name of this that this is referring to a migration of row encoding. I'd suggest either adding some detail to the name about that or at least a comment indicating it's intent. Otherwise it seems a lot more general than what appears to be a one-time migration.